tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/ILoggerTests.cs

```diff
@@ -36,13 +36,18 @@ public ILoggerTests(ITestOutputHelper output)
         [Trait("RunOnWindows", "True")]
         public void InjectsLogs()
         {
+#if NETFRAMEWORK
```

Comment:
I don't get why it's different depending on the framework, could you enlighten please?

Reply:
That's a good question. The sample hosts an AspNetCore server and sends a request to it. So on .NET Core this ends up in one distributed trace with the AspNetCore automatic instrumentation connecting the spans. On .NET Framework we do not instrument AspNetCore so instead of one distributed trace, we end up with two separate traces. I'll add this as a code comment 👍🏼

Reply:
Comment added in 220bd64a22385aa2120912bad4b922c35a8e8de7